docs/2022-07-18-JSON-RPC-Server-API.md

```diff
@@ -292,9 +293,32 @@ where `original-term`, `original-term-index`, `substitution` and `rewritten-term
   }
 }
 ```
-
 where `rewritten-term` is optional and `origin` is one of `kore-rpc`, `booster` or `llvm`.
 
+When `log-timing` is set to `true`, the response to `execute`, `simplify` and `implies` requests will contain information about the wall-clock time spent in different components (`kore-rpc`, `booster`, `proxy`) or overall (without qualifying `component`) while processing the request.
+
+```json
+{
+  "tag": "processing-time",
+  "time": 69.12
+},
+{
+  "tag": "processing-time",
+  "component": kore-rpc",
+  "time": 12.34
+},
+{
+  "tag": "processing-time",
+  "component": "booster",
+  "time": 56.78
+}
+```
+The number in the `time` field  of an entry represents time in seconds.
+
+When no component is given, the time (`69.12` in the example) is the overall wall-clock time spent processing the request, including the given component timings (which are optional). There should be at most one log object without `component` in the list of `logs`. In contrast, log entries with `component` may be repeated (in case there were several entries into the component).
```

Comment:
Will this invariant hold in both kore-rpc and proxy? I.e. does `kore-rpc` emit the empty message when being called via the proxy?

Reply:
Ah ok, looking at the PR, it seems kore-rpc always emits the component field